### ISSUE TITLE
Make remote repo functionality more apparent to Enterprise users

### ISSUE THREAD

**Issue by taylorsperry** [sourcegraph/jetbrains#1149]
Users don't understand that they need to manually add remote repos to give to Cody as chat context or how to do it. 

@toolmantim has created some [designs](https://www.figma.com/file/jEsnzgsf0hNuJbqB9pKaGm/VS-Code---Cody---Commands-up-front-and-ChatGPT%2B%2B?type=design&node-id=1-52&mode=design&t=N6HaPsgc7Oa1N7zg-0) to improve discoverability in VSC. Let's work from that. 

As for ease of adding repos, we have some guidance for typing in a GitHub url, but not other code hosts. Until we can make it so the user just starts typing something like "cody" and the input autofills with "github.com/sourcegraph/cody" and/or a dropdown of indexed repos to select from, we need to at least help non-GH users type in the right thing. That's blocked on https://github.com/sourcegraph/jetbrains/issues/605. 
<img width="502" alt="Screenshot 2024-03-20 at 8 51 00 AM" src="https://github.com/sourcegraph/jetbrains/assets/43555476/63b6ed9f-a36a-4387-a45e-42e8e9a5eb74">

**Comment by taylorsperry** [comment]
Closing in favor of https://github.com/sourcegraph/jetbrains/issues/982
